server/pbench/bin/gold/test-7.8.txt

```diff
@@ -1039,7 +1039,19 @@ len(actions) = 19
         },
         "_type": "pbench-tool-data-iostat"
     }
-]
+]could not convert binary: /var/tmp/pbench-test-server/tmp/uperf__2016-10-06_16:34:03/1-tcp_stream-16384B-32i/sample1/tools-default/flat8_40gb/sar/sar.data
```

Comment:
Let's get this sa binary file issue resolved.  And we have to do it with out carrying around binary, pre-compiled versions of the sadf commands in the tree.

I think we need to tell folks how to name them and place them on a certain path to get that compatibility, but in our tree just use one from the environment.